pom.xml

```diff
@@ -17,13 +17,28 @@
 
 	<properties>
 		<java.version>1.8</java.version>
+		<!-- following two settings fixed 500 error, along with setting tomcat starter to 2.07 -->
+		<servlet-api.version>3.0.1</servlet-api.version>
+<!--		<tomcat.version>8.5.60</tomcat.version>-->
```

Comment:
User-facing comments are fine, but let's remove any commented-out dependencies/properties.

Reply:
I removed the unnecessary commented dependencies with the github editor